jira/cli/templates.go

```diff
@@ -52,12 +52,18 @@ comments:
 `
 const default_edit_template = `# issue: {{ .key }}
 update:
+{{if .overrides.comment}}
```

Comment:
I just found a bug that would try to post "\n" for the comment body if you don't modify the empty template field, which i think is causing this issue for you.  I have fixed it with [47ced2f].

Also if you want to see what go-jira would post without actually posting data, you can use `--dryrun --verbose --verbose` which will now print the json data we would have posted, without actually posting the document.  It can be helpful to debug stuff like this.